ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+///Properties that make up cell content
+@objc(MSFListVnextCell)
+public class MSFListVnextCell: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingView: UIImage?
+    @objc @Published public var title: String = ""
+    @objc @Published public var subtitle: String?
+    @objc @Published public var trailingView: TableViewCellAccessoryType = .none
```

Comment:
Maybe name it trailingViewStyle or even accessoryType?
I think we should define our own enumeration values here instead of reusing the TableViewCellAccessoryType.